src/scripts/Makefile

```diff
@@ -38,32 +38,8 @@ CXXFLAGS=-I.
 
 all: charm++ LIBS AMPI f90charm TSM SM PVM everylb
 
-all-test: all-test-tests all-test-examples all-test-benchmarks
 
-all-test-tests: LIBS
-	$(MAKE) -C ../tests all
-
-all-test-examples: LIBS
-	$(MAKE) -C ../examples all
-
-all-test-benchmarks: LIBS
-	$(MAKE) -C ../benchmarks all
-
-all-zc: charm++
-	$(MAKE) -C ../tests/charm++/zerocopy all
-	$(MAKE) -C ../examples/charm++/zerocopy all
-	$(MAKE) -C ../benchmarks/charm++/zerocopy all
-
-all-test-AMPI: all-test-AMPI-tests all-test-AMPI-examples all-test-AMPI-benchmarks
-
-all-test-AMPI-tests: AMPI
-	$(MAKE) -C ../tests/ampi all
-
-all-test-AMPI-examples: AMPI
-	$(MAKE) -C ../examples/ampi all
-
-all-test-AMPI-benchmarks: AMPI
-	$(MAKE) -C ../benchmarks/ampi all
+include ../../cmake/Makefile.tests.common
```

Comment:
I believe this will fail if `make` is called from the `tmp` symlink folder made directly in `charm`. Is it possible to copy or symlink the file to `tmp` under the old build system instead?

Reply:
That's a good point. 
However, on testing this, it works fine. I think that is primarily because `tmp` is symlinked to `<arch>/tmp`, `../../cmake/Makefile.tests.common` is still valid.